roles/jupyterhub/templates/jupyterhub_config.py.j2

```diff
@@ -9,7 +9,11 @@ c.JupyterHub.port = 8000
 c.JupyterHub.config_file = u'{{ jupyterhub_config_dir }}/jupyterhub_config.py'
 c.JupyterHub.cookie_secret_file = u'{{ jupyterhub_srv_dir }}/cookie_secret'
 c.JupyterHub.db_url = u'{{ jupyterhub_srv_dir }}/jupyterhub.sqlite'
-c.JupyterHub.proxy_auth_token = u'{{proxy_auth_token}}'
+# I dont know why, but this messes it up. Says its depricated
+# it seems like it automatically picks an auth token now and chooses a
+# different one each time
```

Comment:
Did you look up the error message? The comment here doesn't offer enough for someone to understand this.

Reply:
Tthe error message was a 401 permission denied status code from the proxy because they didn't have the same auth_token. The auth token is set to a random value each time jupyterhub is started. If we specify the auth token here, it doesn't agree with the random one the proxy has, but if we don't specify anything, they both have the same random one. I think there may have been an update in some software that caused this new behavior. I didn't want to specify all this information right into the config file, and I am not really sure where to put it.

Reply:
If it is a deprecated feature, we should move away from using it. If it is no longer needed, then just delete. The comment can be removed if you delete it.

Reply:
yes, I removed it 6206f4015d63ef6bd96bcc302f07a6145713476c